node/impl/client/client.go

```diff
@@ -1046,113 +1119,39 @@ func (a *API) ClientDealPieceCID(ctx context.Context, root cid.Cid) (api.DataCID
 }
 
 func (a *API) ClientGenCar(ctx context.Context, ref api.FileRef, outputPath string) error {
-	id, st, err := a.imgr().NewStore()
+	id := imports.ID(rand.Uint64())
+	tmp, err := a.imgr().NewTempFile(id)
```

Comment:
Why is this even on the ImportMgr, if it's not managed by it nor it uses an actual ID.

Reply:
Fixed.